### ISSUE TITLE
[Bug] tooltip‘s `valueFormatter` not work

### ISSUE THREAD

**Issue by kkxxkk2019** [VisActor/VChart#3548]
### Version

latest

### Link to Minimal Reproduction

none 

### Steps to Reproduce

自 `1.12.8` 开始不生效。

```ts
const spec = {
  "categoryField": "描述",
  "data": [
    {
      "id": "data",
      "values": [
        {
          "描述": "任务类型",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "作者id",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "作者昵称",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "批量作者昵称",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "一级作者标签",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "二级作者标签",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "三级作者标签",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "级作者标签签",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "格子标签",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "任务id",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "发布时间",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "截止时间",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "实际结束时间",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "负责人",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "申诉状态",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "优质",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "作者竞争态势",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        },
        {
          "描述": "格子竞争态势",
          "count": 1,
          "__SPEC_DATA_ID__": "data_0"
        }
      ]
    }
  ],
  "innerRadius": 0.5,
  "label": {
    "interactive": true,
    "visible": true
  },
  "labelLayout": "region",
  "legends": {
    "_originalVisible": true,
    "autoPage": false,
    "id": "legends_0",
    "interactive": false,
    "orient": "bottom",
    "position": "start",
    "visible": false
  },
  "markArea": [],
  "markLine": [],
  "outerRadius": 0.8,
  "padAngle": 0.2,
  "pie": {
    "state": {
      "hover": {
        "lineWidth": 1,
        "outerRadius": 0.85,
        "stroke": "#000"
      },
      "selected": {
        "lineWidth": 1,
        "outerRadius": 0.85,
        "stroke": "#000"
      }
    },
    "style": {
      "cornerRadius": 10
    }
  },
  "tooltip": {
    "id": "tooltip_0",
    "mark": {
      "content": [
        {
          "key": "计数",
          "valueFormatter": "{count}"
        },
        {
          "key": "百分比",
          "valueFormatter": "{_percent_}%"
        }
      ]
    }
  },
  "type": "pie",
  "valueField": "count"
};

const vchart = new VChart(spec, { dom: CONTAINER_ID });
vchart.renderSync();

// Just for the convenience of console debugging, DO NOT COPY!
window['vchart'] = vchart;
```

### Current Behavior

<img width="904" alt="image" src="https://github.com/user-attachments/assets/5dca907b-5b22-44e4-8894-aa0f6c4b5dc0">

### Expected Behavior

![image](https://github.com/user-attachments/assets/d4217bbc-fc83-460d-a12a-11c8a391c781)

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by kkxxkk2019** [comment]
bugserver： https://bugserver.cn.goofy.app/case?product=chartspace4&fileid=6758f68971189400b2ec8e3c

**Comment by xile611** [comment]
fixed in 1.13.1
